pkg/chaosctl/cmd/debug.go

```diff
@@ -26,7 +26,6 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/chaos-mesh/chaos-mesh/pkg/chaosctl/common"
```

Comment:
In line 28, it has been declared twice, so removing `cm` should not impact, can you explain me little bit? :)

Reply:
for example:

https://github.com/chaos-mesh/chaos-mesh/blob/7e12dea08017445d11f95e5540df5a34d4916ea1/pkg/chaosctl/cmd/debug.go#L76

should be 


```
	clientset, err := common.InitClientSet()
```